### DISCUSSION TITLE
How do you rename a game directory?

### DISCUSSION THREAD

**Discussion by mOctave** [evennia/evennia#3128] [Evennia Questions] [answered]
So, I started off with Evennia planning to make a test game before working on anything else. My plans changed, and so I've ended up using that directory (named "test") as my main directory. How can I rename the directory to something more suitable without breaking everything else?

**Comment by InspectorCaracal** [comment, marked as answer]
You can just rename it! The only thing it might break is your OS shortcuts, if you have any.

**Reply by mOctave** [reply]
I've found just renaming it leads to `evennia restart` not working, but just sitting there for a few minutes (it normally takes max 5 seconds), doing nothing before I kill it.

**Reply by InspectorCaracal** [reply]
....*oh*. Yes, you have to stop the server before renaming the directory, otherwise the server will be unable to find itself to stop the code. You should always stop a piece of software before trying to move it to a new location, whether that's copying the files or renaming the location.

**Reply by mOctave** [reply]
Thanks, that did it!
